roles/omero_server/templates/systemd-system-omero-server-service.j2

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=OMERO.server
+# Requires: forces the dependency to be started
+# After: ensures this service starts after the dependency, but only if the
+# dependency is also started (PostgreSQL may be on a different server)
+After=postgresql-11.service
+After=postgresql-12.service
+After=postgresql-13.service
+After=postgresql-14.service
```

Comment:
not for this PR, but should we add this to the tomcat service config for XNAT so it doesn't start before postgres?